metrics/sacrebleu/sacrebleu.py

```diff
@@ -127,8 +127,8 @@ def _compute(
             smooth_value=smooth_value,
             force=force,
             lowercase=lowercase,
-            tokenize=tokenize,
             use_effective_order=use_effective_order,
+            **(dict(tokenize=tokenize) if tokenize else {}),
```

Comment:
we may want to improve readability ? like ` tokenize = tokenize if tokenize else None`, and then pass `tokenize=tokenize #L131`, would it be compatible ?

Reply:
Unfortunately, that will not work: we cannot pass `tokenize=None` to `scb.corpus_bleu()`.

On the contrary, if our `tokenize` si None, we must not pass it to `scb.corpus_bleu()` (and leave them use their default value).

Take into account that it is not the same passing `tokenize=None` as not passing `tokenize` parameter.

Reply:
With this PR implementation, we only pass `tokenize` parameter if it is not None; if it is None, we do not pass it.